test/e2e_node/kubelet_test.go

```diff
@@ -204,7 +204,12 @@ func createPod(f *framework.Framework, podName string, containers []api.Containe
 			// Don't restart the Pod since it is expected to exit
 			RestartPolicy: api.RestartPolicyNever,
 			Containers:    containers,
-			Volumes:       volumes,
+			SecurityContext: &api.PodSecurityContext{
+				SELinuxOptions: &api.SELinuxOptions{
+					Level: "s0:c123,c456",
```

Comment:
Are these references specific to a particular SELinux setup, or valid for all SELinux setups? Are they documented anywhere?